MAVProxy/mavproxy.py

```diff
@@ -930,9 +927,9 @@ def run_script(scriptfile):
     if os.path.exists("/usr/sbin/ModemManager"):
         print("WARNING: You should uninstall ModemManager as it conflicts with APM and Pixhawk")
 
-    if opts.mav09:
```

Comment:
I think we should try to preserve the old arguments for a time.  There are going to be people out there with scripts using the old arguments.

I suggest a subroutine `set_mav_version` or similar, taking opts as an argument.

Passing mav10 and mavversion at the same time should be fatal.  Passing mav09/mav10 should issue a warning.